src/components/editor/Editor.jsx

```diff
@@ -33,16 +33,25 @@ class Editor extends Component {
   }
 
   render() {
+    console.warn(this.props)
     return(
       <div id="editor">
         <Header triggerEditorMenu={this.props.triggerHandleOffsetMenu}/>
-        <h1>[Clone|Edit] title.of.resource</h1>
-        <StartingPoints
-          tempStateCallback={this.resetTempState}
-          resourceTemplatesCallback={this.setResourceTemplates}
-          defaultRtId = {this.state.resourceTemplateId}
-          setResourceTemplateCallback={this.setResourceTemplates}
-        />
+        <div className="row">
+          <section className="col-md-9">
+            <h3>Resource Template Label</h3>
+            <h1>[Clone|Edit] <em>Name of Resource</em></h1>
```

Comment:
i'm ok with this, but I wanted to mention that I used "Title" because it is actually the element in question.  Do we have a ticket to fix this to display the actual value?

Reply:
Yes ticket [#393](https://github.com/LD4P/sinopia_editor/issues/393)